### PR TITLE
feat: added FamilyGroup.cs FamilyGroupConfiguration.cs

### PR DESCRIPTION
**Pull request by bramburn**
Update: Refugee.cs to include FamilyGroup.cs parent

#33

### CODE REVIEW

**Comment by Spectralgo** [comment]
@bramburn there are a lot of good things already in there. 

**We should use this has a base for a video.** 

We would go through this PR with you @bramburn. Not only that, but we would reuse most of this code to resubmit from a brand-new branch, respecting The Standard code practices.

We would answer:
- Where to find the list of "Things/Tasks/Issues" you could add to the project ?
- How to fork and create a branch, properly named ?
- What are the architectural conventions ?
- What is the coding style/esthetic we demand ?
- How to make a commit message the right way ?
- How to name your PR ?

@ElbekDeveloper do you have some time next week for a session during an afternoon. It could be a great opportunity to show people how to contribute step by step. **What about next Tuesday (12:00 UTC)** @bramburn ? 

I'm going to do it, but I would love for you or @mabroukmahdhi  to be there.

**Comment by mabroukmahdhi** [comment]
Hello,
I can join.. I hope that @ElbekDeveloper too

**Comment by bramburn** [comment]
fair enough sounds thanks guys!

File: RefugeeLand.Core.Api/Brokers/Storages/IStorageBroker.FamilyGroup.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using RefugeeLand.Core.Api.Models.Family;
+using RefugeeLand.Core.Api.Models.Refugees;
+
+namespace RefugeeLand.Core.Api.Brokers.Storages
+{
+    public partial interface IStorageBroker
+    {
+        ValueTask<FamilyGroup> InsertFamilyGroupAsync(FamilyGroup refugee);
```
Comment (bramburn) [inline RefugeeLand.Core.Api/Brokers/Storages/IStorageBroker.FamilyGroup.cs:16]: keep it empty

**Pull request closed without merging**